src/components/ReserveForm.jsx

```diff
@@ -0,0 +1,100 @@
+import { useRef } from 'react';
+import { useNavigate } from 'react-router-dom';
+
+import '../assets/stylesheets/reservation.css';
+
+export default function ReserveForm({ trip, trips }) {
+  const navigate = useNavigate();
+  const user = JSON.parse(localStorage.getItem('user'));
+
+  const departureCity = useRef('');
+  const timeRef = useRef('');
+  const dateRef = useRef('');
+  const tripIdRef = useRef('');
+
+  const reserveHandler = () => {
+    if (tripIdRef.current.value === '' || departureCity.current.value === ''
+      || timeRef.current.value === '' || dateRef.current.value === '') {
+      // window.alert('Please fill all the fields');
```

Comment:
- [ ] Kindly, remove this all piece of commented code in order to improve your repo's cleanliness. 👍